src/authorizations/users.rs

```diff
@@ -0,0 +1,42 @@
+fn authorized_users() -> Vec<String> {
+    dotenv::var("AUTHORIZED_USERS")
+        .unwrap()
+        .split(',')
+        .map(String::from)
+        .collect()
+}
+
+pub fn is_authorized(name: String) -> bool {
+    authorized_users().contains(&name)
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // The Authorized users environmental variable
+    // is set for tests in the .env file
```

Comment:
Hmm, can we avoid relying on `.env` for tests? It'd be weird to see test failures just because `.env` changed.

Reply:
Well it's not actually dependent on the `.env` for tests. On CI it's set through the GitHub workflow, the `.env` is only for local dev.

Reply:
I'd still prefer if the tests were self-contained and they didn't depend on an environment variable set outside of the process.

Reply:
I can understand the desire, but it feels like we're going against the intent of [dotenv](https://crates.io/crates/dotenv) to set the local testing environmental variable within the test rather than within the .env file.

Reply:
I have clarified the comment a bit more

Reply:
And I'm willing to revisit this in the future if it proves confusing

Reply:
Actually...after breaking my CI tests by forgetting to update the env variable, I think you're right on this one. Will change that today :)

Reply:
I also feel fuzzy about unit tests depending on environment variables in this particular way. It's pretty normal to depend on the environment for e.g. database auth in tests, but this depends on particular values that are only provided in the CI configuration.

If the configuration gets more complex, I'd prefer to split the code initializing the configuration from the code consuming the configuration. Roughly something like this (completely untested):

```
struct Config {
    authorized_users: Vec<String>,
}

impl Config {
    fn new(authorized_users_str: &str) -> Self {
        let authorized_users = authorized_users_str.split(',').map(String::from).collect();
        Config { authorized_users }
    }

    fn is_authorized(id: &str) -> bool {
        self.authorized_users.contains(id)
    }
}

impl Default for Config {
    fn default() -> Self {
        Config::new(
            &dotenv::var("AUTHORIZED_USERS").expect("env variable AUTHORIZED_USERS must be set"),
        )
    }
}
```

This way, everything except for the `Default` implementation can be tested independently from the environment, and the default implementation doesn't really need testing, since it just plugs strings from the environment into the `Config::new()` function.

(If `Config` gets more complex, `new()` should be replaced by a builder.)

But yeah, getting something that works first may be a more important goal at this stage. :)